hil_extension/cli.py

```diff
@@ -0,0 +1,875 @@
+#!/bin/python
+
+# Copyright 2013-2014 Massachusetts Open Cloud Contributors
```

Comment:
Is this copied from the HIL repo?  If so, why can't we just reference from there?  If it isn't, why do we have a copyright license from the MOC?